x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -89,6 +94,33 @@ export const FormContext: React.FC<Props> = ({
     return formData;
   };
 
+  const transformCustomFieldsData = useCallback(
+    (customFields: { [x: `customFields.${string}`]: string | boolean }) => {
+      const transformedCustomFields = [];
+
+      if (!customFields || !customFieldsConfiguration.length || isLoadingCaseConfiguration) {
```

Comment:
I think the validation of the loading should not play a role in the results of the function.